src/main/webapp/resources/js/apis/users/groups.js

```diff
@@ -104,3 +104,13 @@ export function removeMemberFromUserGroup({ groupId, userId }) {
 export function getProjectsForUserGroup(groupId) {
   return axios.get(`${BASE_URL}/${groupId}/projects`).then(({ data }) => data);
 }
+
+export function createUserGroup({ name, description }) {
```

Comment:
Comments

Reply:
Updated in d67ddf17ef3c5c40268ce4405320c918192bcf93